pytx3/setup.py

```diff
@@ -35,6 +35,7 @@
     packages=find_packages(exclude=["tests"]),
     install_requires=[
         "python-Levenshtein",
+        "requests",
```

Comment:
Per comment above, once this switches over to urllib, go ahead and remove the requests requirement.